.changes/generate-ed25519-addresses.md

```diff
@@ -0,0 +1,5 @@
+---
+"wallet-nodejs-binding": patch
+---
+
+Rename generateAddresses to generateEd25519Addresses for Account and SecretManager.
```

Comment:
Also added generateEvmAddresses

Reply:
I didn't tho

Reply:
Actually that's still missing in the nodejs binding

Reply:
It's required, Firefly uses the nodejs binding

Reply:
yikes, this PR is going to balloon

Reply:
Hopefully good now